standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -544,6 +544,15 @@ loop.standaloneRoomViews = (function(mozL10n) {
         case ROOM_STATES.JOINING:
         case ROOM_STATES.JOINED:
         case ROOM_STATES.SESSION_CONNECTED:
+          var roomName = this.props.room.roomName ? this.props.room.roomName :
```

Comment:
just for the sake of code style, I'd take this to a function, same as `renderContext()`